README.md

```diff
@@ -15,13 +15,17 @@ git clone https://github.com/Data4Democracy/collect-social.git
 Then install the requirements using `pip`. 
 
 ```bash
+cd collect-social
 pip install -r requirements.txt
+pip install .
```

Comment:
No, the only concern in my mind is that any program that imports collect_social must have the collect_social module as a subdirectory. For people following the readme, they must have their script placed in the root directory of the project for collect_social to be importable.